client/my-sites/upgrades/domain-management/add-google-apps/add-email-addresses-card.jsx

```diff
@@ -153,7 +153,7 @@ const AddEmailAddressesCard = React.createClass( {
 				<FormTextInputWithAffixes
 					onChange={ this.handleFieldChange.bind( this, 'username', index ) }
 					onFocus={ this.handleFieldFocus.bind( this, 'Email', index ) }
-					placeholder={ this.translate( 'e.g. contact', { textOnly: true, comment: 'Placeholder example email username: contact@...' } ) }
+					placeholder={ this.translate( 'e.g. %(example)s', { args: { example: 'contact' } } ) }
```

Comment:
I think this needs the same treatment as seen in https://github.com/Automattic/wp-calypso/pull/3409/files#diff-82947b8d3e08fc2df6770d749bf802cbR120